examples/stable-diffusion/training/train_text_to_image_sdxl.py

```diff
@@ -13,12 +13,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""
-Fine-tuning script for Stable Diffusion models for text2image with support for LoRA.
-Adapted from the following sources:
-https://github.com/huggingface/diffusers/blob/v0.23.1/examples/text_to_image/train_text_to_image_sdxl.py
```

Comment:
can you add this to right?

Reply:
You mean adding this comment to our script change? But we don't have any of lora change. We overwrote the original script.